src/pages/showcase/submit.astro

```diff
@@ -0,0 +1,190 @@
+                        <Icon
+                            name="heroicons-outline:check"
+                            size={28}
+                            class="text-green-500"
+                            aria-hidden="true"
+                        />
+                        {text}
+                    </li>
+                ))
+            }
+        </ul>
+    </section>
+
+    <Aside class="max-w-screen-sm text-sm shadow-md w-full mx-auto">
+        <p class="text-justify">
+            <Icon
+                pack="heroicons-outline"
+                name="information-circle"
+                size="1.5em"
+                class="inline-block"
+                aria-hidden="true"
+            />
+            Looking for inspiration for your theme's preview images? Check out our
+            <a
+                href="/resources/image-templates"
+                target="_blank"
+                class="underline font-semibold">Image Templates</a
+            >!
+        </p>
+    </Aside>
+
+    <form
+        name="themeSubmit"
+        method="POST"
+        enctype="multipart/form-data"
+        action="/.netlify/functions/submit-showcase"
+        class="max-w-screen-sm mx-auto my-8 px-4"
+    >
+        <div class="flex flex-col gap-6 items-center">
+            {/* https://docs.netlify.com/forms/spam-filters/#honeypot-field */}
+            <p class="hidden">
+                <label>
+                    Don&apos;t fill this out if you&apos;re human:{' '}
+                    <input name="botField" />
+                </label>
+            </p>
+
+            <Field label="Preview images (max 8MB combined)">
+                <ImageInput
+                    name="images"
+                    label="Preview image"
+                    required
+                    client:idle
+                />
+
+                <div class="grid md:grid-cols-2 w-full gap-4 pt-3">
+                    {
+                        Array.from({ length: 4 }, (_, i) => (
+                            <ImageInput
+                                name="images"
+                                label={'Additional image ' + (i + 1)}
+                                client:idle
+                            />
+                        ))
+                    }
+                </div>
+            </Field>
+
+            <InputField
+                name="authorName"
+                label="Your name"
+                placeholder="Sally Ride"
+                required
+            />
+
+            <InputField
+                name="authorEmail"
+                label="Your email"
+                type="email"
+                placeholder="me@website.com"
+                required
+            />
+
+            <InputField
+                label="Website name"
+                name="websiteName"
+                placeholder="My Awesome Website"
+                required
+            />
+
+            <TextAreaField
```

Comment:
(change from the current form) We never actually used the description, maybe we just remove it now?